src/task7/2_test.py

```diff
@@ -0,0 +1,22 @@
+from digits_multiplication import checkio
+import unittest
+
+
+class CheckioTest(unittest.TestCase):
+
+    def test_valid(self):
```

Comment:
we can use ddt with value, expected value and make 3 tests from this one for example